content/docs/get-started-with-neon/regions.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Regions
+enableTableOfContents: true
+isDraft: true
+---
+Neon supports project deployment in several regions. We recommended that you select the region closest to your application server to reduce latency between your Neon project and your application.
+
+## Available regions
+
+Neon currently supports the following AWS regions:
+
+- Asia Pacific (Singapore) Region &mdash; `ap-southeast-1`
```

Comment:
Currently US is more important to us than APAC, we got to list them in first place.

Reply:
Reordered as suggested.